gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/SettingsGradingSchemaPanel.java

```diff
@@ -300,3 +304,23 @@ public int compare(final String o1, final String o2) {
 		}
 	}
 }
+/**
+ * Comparator to ensure correct ordering of percents
+ */
+class PercentComparator implements Comparator<String>, Serializable {
```

Comment:
@steveswinsburg I still see the LetterGradeComparator inside the panel file, do you want me to move this to a separated file?

Thanks.